packages/core/src/codewhisperer/util/customizationUtil.ts

```diff
@@ -29,15 +36,19 @@ export const getNewCustomizations = (availableCustomizations: Customization[]) =
     return availableCustomizations.filter((c) => !persistedCustomizations.map((p) => p.arn).includes(c.arn))
 }
 
-export async function notifyNewCustomizations() {
+export async function notifyNewCustomizations(client?: LanguageClient) {
```

Comment:
yea let's wrap all these things into a class and have the client in one place instead of passing the client in all function calls